src/macros.js

```diff
@@ -3,6 +3,9 @@
  * This can be used to define some commands in terms of others.
  */
 
+import symbols from "./symbols";
+import utils from "./utils";
+
 // This function might one day accept additional argument and do more things.
 function defineMacro(name, body) {
```

Comment:
Once we get flow in place we'll be able to type this as:
```
function defineMacro(name: string, body: string | () => string) {
```

Reply:
Added.